datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -413,7 +434,7 @@ def apply(self):
                     os.remove(writer.ann_file)
                 continue
 
-            writer.write()
+            writer.write(pool)
```

Comment:
How about just accepting all arguments since `_SubsetWriter` could parents of other than one from `datumaro_binary`?
```suggestion
            writer.write(*args, **kwargs)
```

Reply:
Same here.